app/user/helpers.py

```diff
@@ -24,16 +24,16 @@ def send_password_reset_mail(user):
 Der Link ist für 1 Tag gültig.
 
 Viele Grüße
-Dein ZaPF-Auth-System""".format(
-        user, url
+Dein {2}}""".format(
```

Comment:
Ist es möglich, hier auch f-strings statt .format() zu benutzen?

Reply:
Ja - wobei ich mir nicht sicher war, ob woanders noch Format-Strings aus Gruenden drin sind.

Ich moechte in einem spaeteren Commit eh nochmal das Mail-Versenden ueberarbeiten, und da Templates draus machen, bzw. die Mails konfigurierbar machen, also wuerd ich das hier lieber erst reinmergen und dann spaeter dort fixen.

Reply:
FInde ich gut!